old-fashioned-image-build

```diff
@@ -34,8 +34,8 @@ while :; do
             ;;
         --extra-ppa)
             # inject public ppas into the build environment
-            # e.g. 'lp:mylpname/someppa'
```

Comment:
I had to look up what the correct input was to validate your change here.

I can confirm that the [--extra-ppa option](https://github.com/chrisglass/ubuntu-old-fashioned/blob/master/old-fashioned-image-build#L38) is [passed by old-fashioned-build](https://github.com/chrisglass/ubuntu-old-fashioned/blob/master/old-fashioned-image-build#L225) to launchpad-buildd and that code [calls live-build config](https://bazaar.launchpad.net/~canonical-launchpad-branches/launchpad-buildd/trunk/view/head:/lpbuildd/target/build_livefs.py#L157) with those in the EXTRA_PPAS environment variable.  The lb config in livecd-rootfs takes each EXTRA_PPAS string to [populated apt source lines](https://git.launchpad.net/livecd-rootfs/tree/live-build/auto/config#n1157) in the form `deb http://ppa.launchpad.net/$extra_ppa/ubuntu @DISTRIBUTION@ main`